tests/server/db/submitAssessmentDB.test.js

```diff
@@ -0,0 +1,23 @@
+const env = require('../../test-environment')
```

Comment:
Please also keep the naming of the file the same as you would find it in server/db

Reply:
Done